datafusion/src/optimizer/mod.rs

```diff
@@ -25,4 +25,5 @@ pub mod hash_build_probe_order;
 pub mod limit_push_down;
 pub mod optimizer;
 pub mod projection_push_down;
+pub mod remove_duplicate_filters;
```

Comment:
Probably we should rename it now to something like "simplify expressions".

Reply:
Should we rename it to 'optimizer::simplify_expressions' ?

Reply:
I think that would be a more accurate name by now 👍

Reply:
Ok, changed to simplify_expressions.